server/models/role.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const RoleSchema = new Schema({
+  title: {
+    type: String,
+    unique: true,
+    required: true,
+  },
+
```

Comment:
Consider having a description field for your roles. Will make it easier dealing with edits and deletes.